payloads/payloads_test.go

```diff
@@ -0,0 +1,44 @@
+package payloads
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestExtractRunners(t *testing.T) {
+	if !hasPayloads() {
```

Comment:
Check out `testing/fstest` and `fstest.MapFS` for testing against different `embed.FS` setups